qcodes/instrument_drivers/basel/sp983c.py

```diff
@@ -38,6 +39,13 @@ def __init__(self, name: str, **kwargs: Any):
                            vals=Enum(30., 100., 300., 1e3, 3e3, 10e3, 30e3,
                                      100e3, 1e6))
 
+        self.add_parameter('bias_volt',
+                           initial_value=0,
+                           label="Bias Voltage",
+                           unit='V',
+                           vals=Numbers(-0.1, 0.1),
```

Comment:
these limits, are they dictated by the pre-amp?

Reply:
yes

Reply:
Are we sure this is True for all models. E.g. here there are listed 4 models https://www.baspi.ch/low-noise-high-stab-itov-conv

2 of them have a +-0.1 v offset and 2 of them have a +- 1 v offset

Reply:
Yes, the current implementation uses ranges for two model of SP983c. We are not sure how the scaling happens for SP983c01 models and hence not implemented.